packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2928,7 +2928,28 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       widget.controller.addListener(_didChangeTextEditingValue);
       _updateRemoteEditingValueIfNeeded();
     }
-    if (widget.controller.selection != oldWidget.controller.selection) {
+
+    if (_selectionOverlay != null
+        && (widget.contextMenuBuilder != oldWidget.contextMenuBuilder
+            || widget.selectionControls != oldWidget.selectionControls
```

Comment:
extremely trivial nit and feel free to ignore: if the `||`s are on the right hand side, rather than the left, things line up slightly more clearly

Reply:
I like it 👍